tests/test_product.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from buy_together_app import models
```

Comment:
You could have made much of your code cleaner if you imported the `Product` class explicitly:
```suggestion
from buy_together_app.models import Product
```
Then you could replace `models.Product` with `Product` all throughout this file.

Reply:
Fixed